applications/ExaquteSandboxApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -46,7 +47,12 @@ void AddCustomProcessesToPython(pybind11::module& m)
     py::class_<MetricDivergenceFreeProcess<3>, MetricDivergenceFreeProcess<3>::Pointer, Process>(m, "MetricDivergenceFreeProcess3D")
     .def(py::init<ModelPart&>())
     .def(py::init<ModelPart&, Parameters>())
-;
+    ;
+
+    py::class_<DivergenceProcess, DivergenceProcess::Pointer, Process >
+        (m, "DivergenceProcess")
```

Comment:
Take into account the naming convention issue above.

Reply:
Done.